mne/epochs.py

```diff
@@ -1840,12 +1840,12 @@ def export(self, fname, fmt="auto"):
             self.load_data()
 
             # remove extra epoc and STI channels
-            chs_drop = \
-                [ch for ch in ['epoc', 'STI 014'] if ch in self.ch_names]
-            self.drop_channels(chs_drop)
+            drop_chs = {'epoc', 'STI 014'}
```

Comment:
You're never adding anything to `drop_chs` so I think it would be better if this was just a list instead of a set. Same thing in `mne/io/base.py`.

Reply:
I used a set so that maybe lookups will be faster, but you're probably right, with only two items right now, lists are probably better.